plugin/src/main/java/com/playmonumenta/scriptedquests/quests/components/GuiItem.java

```diff
@@ -207,8 +233,8 @@ public static List<GuiItem> parseItems(int index, ItemStack itemStack) throws Ex
 		NBTCompound sqguiCompound = nbtItem.getCompound(SQGUI_KEY);
 		if (sqguiCompound != null) {
 			NBTCompoundList moreItems = sqguiCompound.getCompoundList(MORE_ITEMS_KEY);
-			for (NBTListCompound item : moreItems) {
-				result.add(new GuiItem(index, NBTItem.convertNBTtoItem(new NBTContainer(item.getCompound()))));
+			for (ReadWriteNBT item : moreItems) {
+				result.add(new GuiItem(index, NBTItem.convertNBTtoItem(new NBTContainer(item.toString()))));
```

Comment:
This looks like that it might have unintended behavior
However if it works ingame, then it is probably safe to ignore

Reply:
All good lol - thanks for taking a look at it!

I think I made this change because of some compatibility issues (after updating to 2.12.0-SNAPSHOT). It worked ingame after that, so :shrug: